x-pack/plugins/session_view/public/components/detail_panel_process_tab/styles.ts

```diff
@@ -27,10 +27,37 @@ export const useStyles = () => {
       textOverflow: 'ellipsis',
     };
 
+    const metadataHostOS: CSSObject = {
```

Comment:
If this style is only being used for host tab, it should be in `detail_panel_process_tab/styles.ts` instead

Reply:
any updates on this?